test/text-area.html

```diff
@@ -110,6 +110,19 @@
               assertAttrCanBeSet(prop, false);
             });
           });
+
+          ['pattern'].forEach(function(prop) {
+            it('should set string attribute ' + prop, function() {
+              assertAttrCanBeSet(prop, 'none');
+            });
+          });
+
+          ['multilinePatternMode'].forEach(function(prop) {
+            it('should set boolean attribute ' + prop, function() {
+              assertAttrCanBeSet(prop, true);
+              assertAttrCanBeSet(prop, false);
+            });
+          });
```

Comment:
There are lots of missing tests:
- test that textarea becomes invalid if input does not match pattern
- test that validation state updates when changing the pattern while the textarea has an existing value
- test that value does not change when setting preventInvalidInput and inputting an invalid character
- test that multiline pattern mode works as expected
- test that validation state updates when changing multiline pattern while textarea has an existing value

Reply:
I think those tests are already there for text area as well, under test/validation.html